activerecord/lib/active_record/associations.rb

```diff
@@ -1499,6 +1504,9 @@ def has_many(name, scope = nil, **options, &extension)
         #   +:inverse_of+ to avoid an extra query during validation.
         # [:strict_loading]
         #   Enforces strict loading every time the associated record is loaded through this association.
+        # [:owner_ensuring_destroy]
+        #   Specifies a instance method name that will be called in the owner when executing the background job to
+        #   delete the associated record to determine if the record should be deleted.
```

Comment:
Same suggestion as above: `Specifies an instance method that will be called by the owner to destroy the associated record. The associated records will be destroyed in a background job.`